schema.go

```diff
@@ -29,7 +21,7 @@ var Schema = schema.NewTypedScopeSchema[*config.Config](
 				nil,
 				nil,
 				nil,
-				schema.PointerTo(util.JSONEncode(pythonGetDefaultPath())),
+				nil,
```

Comment:
should we set this as "schema.PointerTo(util.JSONEncode("python")),", to maintain consistency with the other deployers?

Reply:
I have added this back in with a ref to now just "python"